client/packages/invoices/src/OutboundShipment/DetailView/SidePanel/RelatedDocumentsSection.tsx

```diff
@@ -23,11 +23,7 @@ const RelatedDocumentsSectionComponent = () => {
     let tooltip = t('messages.customer-requisition-created-on', {
       date: d(createdDatetime),
     });
-    if (username && username !== 'unknown') {
-      tooltip += ` ${t('messages.by-user', { username })}`;
-    }
-
-    return tooltip;
+    return (tooltip += ` ${t('messages.by-user', { username })}`);
```

Comment:
Pretty sure this will show as just `by` with no `-` I think that's ok right?

Reply:
The place where it's being called passes in '-' if no username XD